lab/style-script/gss-functions.md

```diff
@@ -1148,6 +1253,70 @@ Outputs:
 
 - `string`
 
+### `IsNumber(value)`
+
+Returns True if the input value is a number, otherwise False.
```

Comment:
If it returns True or False, i would format it as code... if it returns true and false then they are normal words

Reply:
@tonilastre - you don't agree with this or? This is the only change missing...

Reply:
Ups sorry, totally missed this one. I will change it to code formatted values right away.